### ISSUE TITLE
Still maintained? 

### ISSUE THREAD

**Issue by jwillmer** [SQLStreamStore/SQLStreamStore#507]
I see no major development on this project since over a year and would like to know if it is still maintained. Example projects are archived our out dated and the GettingStarted repo is empty. So I'm a little bit concerned to use this library. Maybe there are better options nowadays?

**Comment by AIV-Bert** [comment]
Seems pretty dead

**Comment by yreynhout** [comment]
It's still a viable option and is in active use on more than one software project. I'd say most maintainers are busy with their day jobs. Of course, there are alternatives. What sort of development would you expect - what features are you missing? Or is this merely a cae of low activity being perceived as "this project is dead"?

**Comment by jwillmer** [comment]
Just the low activity and the open pull requests made me question if the project is still maintained.

**Comment by AIV-Bert** [comment]
The big issue for me is the low response on pull request. Off course we don't expect maintainers to quit ther day jobs. But a few months without reaction, is more then 'merely a case of low activity'.

**Comment by carlosrfernandez** [comment]
I agree with the general concern, and it's not just a matter of "missing features", it's just a feeling that there seems to be no activity at for very long chunks time.

The implementation for Sqlite was merged over a year and a half ago, but as far as I can tell (and anybody feel free to correct me if I'm wrong) I don't see a pre-release nuget package with the SqliteStreamStore implementation.

The Postgres one has also been on "beta" for a while now. 

Same feeling applies to the amount of open PR's updating dependencies to newer versions, that are just there "open"...

We want to use SqlStreamStore in production but the doubts are there.

**Comment by damianh** [comment]
Hi all, 

Sorry for the absence. 

I've been giving some serious thought as to the future of this project. It's obvious that I've (and my associates) not been giving it the attention that it needs. I will follow up with a statement shortly and will be looking for your feedback.

Cheers

Damian
